lib/sprockets/media_query_combiner/processor.rb

```diff
@@ -1,15 +1,40 @@
-require 'tilt'
```

Comment:
This should probably `require "sprockets"` so that we are not relying on autoloading for `.choose`